styles.css

```diff
@@ -5,3 +5,30 @@ p {
 a {
   font-family: Roboto;
 }
+.wrapper {
+ margin: auto;
+ width: 80%;
+ }
+.logo, .menu {
+ display: inline-block;
+}
+.logo {
+ margin-right: 63%;
+}
+.header {
+ display: flex;
+ justify-content: space-between;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/IllenzeerAnna28/pullRequest?prid=6364836)